### PR TITLE
Introduce create job action

### PR DESCRIPTION
**Pull request by orlandohohmeier**
Add Chronos create job action – the action will use the undocumented `/scheduled-jobs` endpoint (which will be removed with the next release) for now, to enable the creation of an job including a schedule in one call.

/cc @jfurrow

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @pierlo-upitup

File: src/js/events/__tests__/ChronosActions-test.js [hunk 1]
```diff
@@ -12,6 +12,47 @@ var ChronosActions = require('../ChronosActions');
...
+      spyOn(RequestUtil, 'json');
+      ChronosActions.createJob();
+      this.configuration = RequestUtil.json.calls.mostRecent().args[0];
+    });
+
+    it('calls #json from the RequestUtil', function () {
+      expect(RequestUtil.json).toHaveBeenCalled();
+    });
+
+    it('fetches data from the correct URL', function () {
```
Comment (pierlo-upitup) [inline src/js/events/__tests__/ChronosActions-test.js]: `sends data to the` ?

Comment (orlandohohmeier) [inline src/js/events/__tests__/ChronosActions-test.js]: Yeah... :)

**Comment by pierlo-upitup** [comment]
I LIEK IED

**Comment by orlandohohmeier** [comment]
Added the missing store config...

File: src/js/events/__tests__/ChronosActions-test.js [hunk 2]
```diff
@@ -12,6 +12,47 @@ var ChronosActions = require('../ChronosActions');
...
+        var action = payload.action;
+        AppDispatcher.unregister(id);
+        expect(action.type)
+          .toEqual(ActionTypes.REQUEST_CHRONOS_JOB_CREATE_SUCCESS);
+      });
+
+      this.configuration.success([]);
+    });
+
+    it('dispatches the correct action when unsucessful', function () {
```
Comment (kennyt) [inline src/js/events/__tests__/ChronosActions-test.js]: `unsuccessful` 👍 two `c`s

**Comment by kennyt** [comment]
:ship: after typo

**Comment by kennyt** [comment]
conflicts

**Pull request merged**